src/test/scala/com/workday/warp/persistence/mysql/WarpSlickSingleColumnExtensionsSpec.scala

```diff
@@ -44,11 +47,12 @@ class WarpSlickSingleColumnExtensionsSpec extends WarpJUnitSpec with CorePersist
     }
 
     val rows: Seq[(Int, Option[Double])] = this.persistenceUtils.runWithRetries(action.result, 5).sortBy(_._1)
-
     rows.head._2 shouldBe defined
     rows.head._2.get should equal (STD_1)
     rows(1)._2 shouldBe defined
     rows(1)._2.get should equal (STD_2)
+    rows(2)._2 shouldBe defined
+    rows(2)._2.get should equal (0)
```

Comment:
Just a minor thing, there's a mix of scalatest matchers here. I see both `should equal (...)` and `shouldEqual` in here. I can't remember which one we prefer, since I have no strong opinion on it, so I'd just picking the more common one and being consistent with it.

I think @tomnis has an opinion on which one to use though?

Reply:
i personally like using `should be`, but its a small enough nit that either `should be` or `shouldBe` is fine.

please take a moment to review the semantics of different equality checks
http://www.scalatest.org/user_guide/using_matchers#checkingEqualityWithMatchers